php/Modules/Photo.php

```diff
@@ -844,8 +844,8 @@ public function getArchive() {
 		}
 
 		// Get extension
-		$extension = getExtension($photo->url);
-		if ($extension===false) {
+		$extension = getExtension($photo->url, true);
```

Comment:
The field is called URL, but it just contains the filename (e.g. 4a3a4b31c88b420c29a007a8db6dd42f.gif). Should be changed to `getExtension($photo->url, false)`, right?